server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/change/StateChange.scala

```diff
@@ -61,7 +61,8 @@ case class StateChangeEvent(eventId: EventId,
       success => success) ->
       TypeState(
         MailboxTypeName.asMap(mailboxState) ++
-          EmailTypeName.asMap(emailState))))
+          EmailTypeName.asMap(emailState))),
+      None)
```

Comment:
Here it should not be 'None'... We should likely compute the global state of the server.

Maybe the 'globalState' is a compulsory property that we want to add to the SateChangeEvent?